uwblueprint/project-read-frontend
# [TS migration] Move fields context to Typescript 📦

### Notion ticket link



[TS migration](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=e47e78d6b54e4120aa13f06bb61640c5)



### Implementation description

- added type annotations to /context/fields.jsx



### Steps to test

1. `yarn start`
2. open the reg table → it should have data for dynamic fields (DOB, internet access, etc.)



### What should reviewers focus on?

- ts best practices

### Checklist
